CONTRIBUTING.md

```diff
@@ -40,16 +46,18 @@ Please follow these steps to make your first PR!
 
     $ make format
     ```
-6. If any of the commands above fail, make changes to your code to successfully pass the three comamnds above.
-7. Once your changes pass the automated tests and formatters, add your changes to the branch and make a commit with a descriptive message:
+6. If any of the commands above fail, make changes to your code to successfully pass the three commands above.
+7. Once your changes pass the automated tests and formatters, add your changes to the branch. Using `git status`, double check that the files are the files that you want to commit. Then make a commit with a descriptive message
     ```
     $ git add .
 
+    $ git status
+
     $ git commit -m "Adding a new feature to Chipy.org"
     ```
 8. Push your commit(s) to your repository fork:
     ```
     $ git push origin BRANCH_NAME
     ```
-9. Create a pull request by going to your repository fork on GitHub and clicking "Compare & pull request)
+9.  Create a pull request by going to your repository fork on GitHub and clicking "Compare & pull request)
```

Comment:
Line 62, please take out extra space before 'Create'. There should be a single space between '9.' and 'Create'. Also, please change the ')' to be a `"` at the end of this line.

Reply:
Fixed 👍🏼